Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/package.json

```diff
@@ -9,20 +9,18 @@
   },
   "devDependencies": {
     "@babel/core": "^7.1.6",
+    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
+    "@babel/plugin-transform-react-jsx": "^7.2.0",
     "@babel/preset-env": "^7.1.6",
     "@babel/preset-react": "^7.0.0",
+    "@dnnsoftware/dnn-react-common": "^2.0.6",
     "array.prototype.find": "2.0.4",
     "array.prototype.findindex": "2.0.2",
-    "babel-core": "6.26.3",
-    "babel-loader": "8.0.4",
-    "babel-plugin-transform-object-assign": "6.22.0",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-plugin-transform-react-remove-prop-types": "0.4.20",
-    "babel-polyfill": "6.26.0",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.21",
     "css-loader": "1.0.1",
     "dnn-global-styles": "0.0.5",
```

Comment:
@dnnsoftware/dnn-react-common has everything, so dnn-global-styles and dnn-webpack-externals should be removed